cmake/README.md

```diff
@@ -15,6 +15,7 @@ The following variables are provided for oneDPL configuration:
 | ONEDPL_FPGA_STATIC_REPORT    | BOOL   | Enable the static report generation for the FPGA_HW device type                               | OFF           |
 | ONEDPL_USE_AOT_COMPILATION   | BOOL   | Enable ahead-of-time compilation for the GPU or CPU device types                              | OFF           |
 | ONEDPL_ENABLE_SIMD           | BOOL   | Enable SIMD vectorization by passing an OpenMP SIMD flag to the compiler if supported; the flag is passed to user project compilation string if enabled | ON           |
+| ONEDPL_WIN_ICPX_ICX_FIXS     | BOOL   | Enable icpx, icx, icx-cl workarounds to fix issues in CMake for Windows.                      | ON            |
```

Comment:
Perhaps I do not understand the problem well, but why do we want to provide an opt-in macro to enable the workarounds, instead of doing it by default (and maybe provide an opt-out macro if the workarounds hurt more than help)? If it was already discussed, feel free to point me out to that discussion.

Generally, I would prefer to only add configuration parameters that are absolutely necessary.

Reply:
This defaults to `ON`, so it is done by default.  I would be fine changing the directionality of the flag, `ONEDPL_SKIP_WIN_ICPX_ICX_FIXES` which defaults to `OFF`.  I like providing the option for someone to turn off the inclusion of workarounds in case they have their own compiler setup modifications which clash with ours.

This option is also only present for Windows hosts using [cmake_dependent_option](https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html).

Reply:
If the flag only affects our tests, let's name and describe it properly. See the line 20 for an example of a test-only configuration parameter.

Reply:
Ah yes, thank you.  I'll make that change.